src/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -222,6 +153,12 @@
     <Compile Include="Microsoft\XmlSerializer\Generator\XmlSerializationReader.cs" />
     <Compile Include="Microsoft\XmlSerializer\Generator\XmlSerializationWriter.cs" />
     <Compile Include="Sgen.cs" />
+    <Compile Include="System\Xml\Serialization\SoapAttributesExtensions.cs" />
```

Comment:
The file is for SGen only. Should the file be under `Microsoft\XmlSerializer\Generator`?